app/models/lockstep/magic_link.rb

```diff
@@ -0,0 +1,10 @@
+class Lockstep::MagicLink < Lockstep::ApiRecord
```

Comment:
Could you add test cases for this? We wouldn't want it to break as other devs contribute to the gem

Reply:
Sure will add it